server/src/main/java/io/seata/server/storage/redis/store/RedisTransactionStoreManager.java

```diff
@@ -92,20 +97,58 @@ public static RedisTransactionStoreManager getInstance() {
         return instance;
     }
 
+
+    /*
+    Map for LogOperation Global Operation
+    */
+    public static volatile ImmutableMap<LogOperation, Function<GlobalTransactionDO, Boolean>> globalMap;
```

Comment:
Why not use one map instead of two?

Reply:
in this class，two map Fuction<T,R> has diff T

Reply:
ok, please fix another review.